Marlin/Configuration.h

```diff
@@ -888,11 +888,12 @@
 
 /**
  * Z Probe to nozzle (X,Y) offset, relative to (0, 0).
- * X and Y offsets must be integers.
- *
+ * Default X and Y offsets must be integers.
+ * Floats may be set with M851 if required.
```

Comment:
For the sake of clarity I think we need to find another word besides "floats." Only geeks know that usage.

Reply:
By that argument integer should be changed as well. Should we go for Whole Numbers / decimals?

Reply:
The word "integer" has a more colloquial usage in general mathematics which all kids are taught. But the word "float" is only used in computer science, and is a nick-name for "floating-point."